json_serializable/test/json_serializable_test.dart

```diff
@@ -188,6 +188,14 @@ abstract class _$OrderSerializerMixin {
       });
     }
 
+
+    test('class with fromJson() constructor with optional parameters', () async {
+      var output = await runForElementNamed('FromJsonOptionalParameters');
+      print(output);
```

Comment:
Remove this guy!

Reply:
oh man... fixed.

I decided to squash the changes to hide my shame...